src/tests/GC/Stress/Framework/RFLogging.cs

```diff
@@ -79,7 +79,6 @@ private void LogWorker()
                 }
                 catch (IOException e)
                 {
-                    ReliabilityFramework.MyDebugBreak(String.Format("LogWorker IOException:{0}", e.ToString()));
```

Comment:
Let's throw the exception here so that we can take a dump

Reply:
If the debugger break is set to true, we should set a debug break point here and if not, log and throw an exception.